crates/typstyle-core/src/attr.rs

```diff
@@ -158,7 +158,9 @@ impl AttrStore {
             // no format nodes with @typstyle off
             if child_kind != SyntaxKind::Space && disable_next {
```

Comment:
should we move the hash check here?

Reply:
this means `#` can be format-disabled. move hash check to the condition is also ok.